src/resources/breaking-changes.md

```diff
@@ -0,0 +1,818 @@
+---
+title: Breaking changes
+description: A list of breaking changes by release in Dart.
+---
+
+This page lists the breaking changes in Dart, organized by release and area.
+Complete release notes are available in the [Dart SDK changelog][changelog].
+
+According to Dart's [breaking change policy][], each listed change contains a link to more
+information on the change, such as migration guidance, background, reasoning,
+caveats, and more.
+
+{{site.alert.note}}
+  Changes prior to Dart 3.1.0 and Dart's updated breaking change policy 
+  might not have links to additional information. 
+  More details on some of those changes might
+  be available in their corresponding [changelog][] entry.
+{{site.alert.end}}
+
+**Dart has two types of breaking changes**:
+* **Standard**: Causes breaks as soon as you [upgrade your sdk version][sdk]. 
+  *These are the majority of changes and are not specially marked in this list.*
+* **Language versioned**: Causes breaks only when you upgrade the
+  [language version] of your code. *These are marked "**Language versioned**"*.
+
+If you have questions or concerns about any of these breaking changes, please 
+[file an issue][]. To be notified about future breaking changes,
```

Comment:
Perhaps say "comment on the breaking change issue linked from the relevant entry" to avoid people filing new issues?